replication-api-impl/src/main/java/org/codice/ditto/replication/api/impl/ReplicatorStoreFactoryImpl.java

```diff
@@ -45,9 +53,9 @@
 
   private static final Integer POLL_INTERVAL_MINUTES = 5;
 
-  private static final Integer CONNECTION_TIMEOUT = 30000;
+  private static final Integer DEFAULT_CONNECTION_TIMEOUT = 30;
```

Comment:
✏️ Please documents that they are in seconds or add something like `_IN_SECS` to the name.